worker/uniter/manifold.go

```diff
@@ -124,7 +127,12 @@ func Manifold(config ManifoldConfig) dependency.Manifold {
 				return nil, err
 			}
 
-			downloader := charms.NewCharmDownloader(apiConn)
+			var s3Caller s3caller.Session
+			if err := ctx.Get(config.S3CallerName, &s3Caller); err != nil {
+				return nil, err
```

Comment:
nit:

```suggestion
				return nil, errors.Trace(err)
```

Reply:
@SimonRichardson do we need to error trace this `err`? I see all other `ctx.Get()` in the manifold returning the error as is

Reply:
Yes please.

Reply:
fixed